.gitignore

```diff
@@ -1,20 +1,21 @@
 # dependencies
-source/node_modules/
+
 /.pnp
 .pnp.js
-.idea/
-.vscode/
-source/node_modules/
+ource/node_modules/
```

Comment:
I think it should be source/node_modules?

Reply:
Good catch. Fixed!